platform/security/core/security-core-services/src/main/java/ddf/security/impl/SubjectIdentityImpl.java

```diff
@@ -34,9 +41,21 @@
    */
   @Override
   public String getUniqueIdentifier(Subject subject) {
-    Optional<String> owner = getSubjectAttribute(subject).stream().findFirst();
-    if (owner.isPresent()) {
-      return owner.get();
+    SortedSet<String> subjectAttribute = getSubjectAttribute(subject);
```

Comment:
✏️ I was a little confused by the `getSubjectAttribute` method. It might be useful to rename that method to something like `getSubjectIdentityAttribute`.

Reply:
I don't disagree, but it's like this in the later DDF versions too, so I'm going to keep it as-is for consistency